client/cypress/end-to-end/ballot-comparison-audit-board.spec.js

```diff
@@ -0,0 +1,121 @@
+before(() => cy.exec('./cypress/seed-test-db.sh'))
+
+describe('Audit Boards', () => {
+  const uuid = () => Cypress._.random(0, 1e6)
+  const id = 0
+
+  before(() => {
+    id = uuid()
+    cy.visit('/')
+    cy.loginAuditAdmin('audit-admin-cypress@example.com')
+    cy.get('input[name=auditName]').type(`TestAudit${id}`)
+    cy.get('input[value="BALLOT_COMPARISON"]').check({ force: true })
+    cy.findByText("Create Audit").click()
+    cy.wait(1000)
```

Comment:
I wouldn't expect you to need to call `wait`, since Cypress has built in retries/timeouts. I think if the default timeout (4 seconds) is too short for some of these checks, it would be better to increase the timeout for those checks rather than add a `wait`, since `wait` makes the test always wait that long even if it's not necessary, whereas a timeout will let the check finish as soon as possible. 

More on that here: https://docs.cypress.io/guides/core-concepts/introduction-to-cypress.html#Timeouts

Another option that I just found out about for this specific case would be to intercept the request for the new route and wait on that: https://docs.cypress.io/guides/guides/network-requests.html#Waiting

Reply:
Yes, I'm trying to minimize their usage. There are a few places where things fail without them that still perplex me, though, and I'm working on figuring them out. I just went through and cleaned up a bunch of extra ones though.

Thanks for pointing out the network request interceptions! That's a really cool feature, and I'll have to take a look and see if they can help resolve some of these.